src/qt/intro.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <interfaces/node.h>
 #include <util/system.h>
+#include <validation.h>
```

Comment:
77f3f2dda6918b0986f785b3061cfb8c97ed2b5c

It's unfortunate to include this just because of `MIN_DISK_SPACE_FOR_BLOCK_FILES`.